core-bundle/assets/styles/jobs/menu.pcss

```
@@ -0,0 +1,44 @@
+*[data-controller="contao--jobs"] {
+    #running-jobs {
+        display: none;
+    }
+
+    /* Show an overlay on hover if there is at least one running job. */
+    &:not(&:has(*[data-contao--jobs-target="count"]:empty)):hover #running-jobs {
```

```suggestion
[data-controller="contao--jobs"] {
    #running-jobs {
        display: none;
    }
    
    &:has([data-contao--jobs-target="count"]:empty) #running-jobs {
        display: none;
    }

    /* Show an overlay on hover if there is at least one running job. */
    &:hover #running-jobs {
```

1. The wildcard is not needed for the attribute selector

2. The second change is simplifying the selector as specificity for the `:has` will be higher than the hover, thus always being `display:none` if there is no count.